views/documentation.pug

```diff
@@ -0,0 +1,18 @@
+extends layout.pug
+
+block head
+	.navbar
```

Comment:
can I make it a separate file and include it in demo/doc/settings? or override will also be fine I guess, which one do you want to go with?